src/components/DateInput.jsx

```diff
@@ -79,6 +79,10 @@ const defaultProps = {
 
   // accessibility
   isFocused: false,
+  ariaLabel: undefined,
+  ariaInvalid: undefined,
+  ariaDescribedby: undefined,
+  screenReaderMessage: '',
```

Comment:
This isn't part of your change, but I wonder if this should default to `undefined` instead of an empty string